cms/djangoapps/contentstore/tests/test_contentstore.py

```diff
@@ -1041,6 +1041,17 @@ def test_course_access_groups_on_delete(self):
         self.assertFalse(instructor_role.has_user(self.user))
         self.assertEqual(len(instructor_role.users_with_role()), 0)
 
+    def test_create_course_after_delete(self):
```

Comment:
Does this test fail if you remove the change to `common/lib/xmodule/xmodule/modulestore/mixed.py`?